src/sentry/search/events/datasets/spans_indexed.py

```diff
@@ -73,6 +73,16 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     result_type_fn=self.reflective_result_type(),
                     default_result_type="duration",
                 ),
+                SnQLFunction(
+                    "avg",
+                    optional_args=[
+                        with_default("span.duration", NumericColumn("column", spans=True)),
```

Comment:
Might be surprising to folks that `avg()` here is `"span.duration"` but for metrics it's `"span.self_time"`

Reply:
Yeah going to do another pass to update all the defaults in indexed to `span.self_time` and add tests 😬